contrib/epee/src/net_ssl.cpp

```diff
@@ -527,7 +528,7 @@ void ssl_options_t::configure(
       // preverified means it passed system or user CA check. System CA is never loaded
       // when fingerprints are whitelisted.
       const bool verified = preverified &&
-        (verification != ssl_verification_t::system_ca || host.empty() || boost::asio::ssl::rfc2818_verification(host)(preverified, ctx));
+        (verification != ssl_verification_t::system_ca || host.empty() || boost::asio::ssl::host_name_verification(host)(preverified, ctx));
```

Comment:
This change should fail on Boost 1.66-1.73. A `#ifdef` will be needed to differentiate between the two names.

Reply:
Fixed.